packages/connect-query/package.json

```
@@ -55,6 +57,8 @@
     "react-dom": "^18.3.1"
   },
   "files": [
-    "dist/**"
+    "dist/**",
+    "!*.test.js",
+    "!*.test.d.ts"
```

Do we still need those?

```suggestion
    "dist/**"
```